src/orca/services/nextflowtower/ops.py

```diff
@@ -130,13 +132,20 @@ def launch_workflow(
         if not ignore_previous_runs:
             latest_run = self.get_latest_previous_workflow(launch_info)
             if latest_run:
+                status = latest_run.status.value
+                run_repr = f"{latest_run.run_name} ({latest_run.id})"
                 # Return ID for latest run if ongoing, succeeded, or cancelled
-                skip_statuses = {"SUCCEEDED", "CANCELLED"}
-                if not latest_run.is_done or latest_run.status.value in skip_statuses:
+                skip_statuses = {"SUCCEEDED"}
```

Comment:
Should this include "CANCELLED"?

Reply:
I had included `CANCELLED` originally based on our conversation yesterday, but when I cancelled a workflow today for the demo, I had to delete that run to allow the script to work. So, maybe I'm not understanding why it should be included. How is it different from `FAILED`?

Reply:
I guess I initially thought that a person's intention in canceling a workflow would be not to run that workflow at all, but it could definitely be that they realize they configured something incorrectly or something where resuming would make sense. So on second thought, I agree with the decision to remove `CANCELLED` from this dictionary

Reply:
Thinking about this further, we might actually want to include `UNKNOWN` in this list because it might represent a workflow that's still running on AWS Batch, so we wouldn't want to relaunch it.